wordpress-mobile/WordPress-Android
# Update Story intro screen assets and links

Updates the Intro to Stories screen:

* New, generic URLs for the intro/example story links (the posts themselves are a WIP right now, but the URLs won't change)
* Updated the preview images to reflect the new posts

![stories-new-intro](https://user-images.githubusercontent.com/9613966/100964842-c901a380-356c-11eb-9b30-872f149802e5.png)

cc @megsfulton 

### To test:

* Clear data so you'll be shown the Introducing Story Posts screen, in case you've already dismissed it
* Check that the two story images show up correctly
* Check that the links are valid

PR submission checklist:
